packages/store/src/lib/flags.ts

```diff
@@ -15,12 +15,13 @@ export const storeFlags = {
 
 export const fileFlags = {
   'from-file': Flags.string({
-    description: 'The SQLite file to import data from.',
+    description: "The SQLite file to import data from. Can't be used with --to-file.",
     required: false,
     env: 'SHOPIFY_FLAG_FROM_FILE',
   }),
   'to-file': Flags.string({
-    description: 'The SQLite file path to export data to. Omit to auto-generate filename.',
+    description:
+      "The SQLite file path to export data to. Omit to auto-generate filename. Can't be used with --from-file.",
```

Comment:
More tiny nits on preposition order: 
```suggestion
      "The SQLite file path to which to export data. Omit to auto-generate filename. Can't be used with --from-file.",
```

Reply:
Alternative suggestion to match the identity key description: 
```suggestion
      "The SQLite file path to use to export data. Omit to auto-generate filename. Can't be used with --from-file.",
```